src/Bulk/Result/BulkResult.php

```diff
@@ -22,8 +22,8 @@
 
 namespace OAT\SimpleRoster\Bulk\Result;
 
-use OAT\SimpleRoster\Bulk\Operation\BulkOperation;
 use JsonSerializable;
+use OAT\SimpleRoster\Bulk\Operation\BulkOperation;
```

Comment:
Just to check: I understand that most probably it was "auto-changed", but if no - what's the reason to do such change?

Reply:
It's done by the IDE, the feature is called "Optimizing imports".